test-case/check-alsabat.sh

```diff
@@ -42,6 +42,9 @@ OPT_HAS_ARG['n']=1             OPT_VAL['n']=240000
 OPT_NAME['s']='sof-logger'      OPT_DESC['s']="Open sof-logger trace the data will store at $LOG_ROOT"
 OPT_HAS_ARG['s']=0             OPT_VAL['s']=1
 
+OPT_NAME['z']='cavstool'       OPT_DESC['z']="Zephyr native logging tool, the data will store at $LOG_ROOT"
+OPT_HAS_ARG['z']=0             OPT_VAL['z']=1
```

Comment:
Why do we need to make this configurable, do we have multiple names to support? Don't submit code that we don't need yet.

https://www.martinfowler.com/bliki/Yagni.html